Pod/Classes/PINRemoteImageManager.m

```diff
@@ -1520,6 +1522,11 @@ - (NSString *)cacheKeyForURL:(NSURL *)url processorKey:(NSString *)processorKey
 
 - (void)objectForKey:(NSString *)key options:(PINRemoteImageManagerDownloadOptions)options completion:(void (^)(BOOL found, BOOL valid, PINImage *image, id alternativeRepresentation))completion
 {
+    if (options & PINRemoteImageManagerDownloadOptionsIgnoreCache) {
```

Comment:
total nit, but do you mind matching the style above, wrapping this in parens and checking to see that it != 0 ?